is_core/auth_token/default/resource.py

```diff
@@ -52,7 +52,7 @@ def __init_core__(cls, core, pattern):
         cls.core = core
         cls.pattern = pattern
 
-    def has_delete_permission(self, obj=None, via=None):
+    def has_delete_permission(self, **kwargs):
```

Comment:
tady se to jemně komplikuje navíc nechápu proč se do předka posílal self.request :D